recipes/cuda-pathfinder/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: cuda-pathfinder
+  version: "1.0.0"
+  # no-arch packages can't use selectors, so we avoid using the "match(python, "<3.9")" selector this way
+  python_min: "3.9"
```

Comment:
Do we want to redefine this or can we rely on the upstream pinning? https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L824

Reply:
Oh I forgot there's a global pinning for Python min too. That said, CFEP-25 does not specify the timeline of deprecating the oldest Python, and it might mismatch what we need. Since I learned this trick from other recently merged Python recipes, I think it's good to keep it for flexibility, but I can also remove it if you like.